examples/python/lattice/simple/build-xml.py

```diff
@@ -0,0 +1,166 @@
+import openmc
+
+###############################################################################
+#                      Simulation Input File Parameters
+###############################################################################
+
+# OpenMC simulation parameters
+batches = 20
+inactive = 10
+particles = 10000
+
+
+###############################################################################
+#                 Exporting to OpenMC materials.xml file
+###############################################################################
+
+# Instantiate some Materials and register the appropriate Nuclides
+fuel = openmc.Material(material_id=1, name='fuel')
+fuel.set_density('g/cc', 4.5)
+fuel.add_nuclide('U235', 1.)
+
+moderator = openmc.Material(material_id=2, name='moderator')
+moderator.set_density('g/cc', 1.0)
+moderator.add_element('H', 2.)
+moderator.add_element('O', 1.)
+moderator.add_s_alpha_beta('c_H_in_H2O')
+
+# Instantiate a Materials collection and export to XML
+materials_file = openmc.Materials([moderator, fuel])
+materials_file.export_to_xml()
+
+
+###############################################################################
+#                 Exporting to OpenMC geometry.xml file
+###############################################################################
+
+# Instantiate Surfaces
+left = openmc.XPlane(surface_id=1, x0=-2, name='left')
+right = openmc.XPlane(surface_id=2, x0=2, name='right')
+bottom = openmc.YPlane(surface_id=3, y0=-2, name='bottom')
+top = openmc.YPlane(surface_id=4, y0=2, name='top')
+fuel1 = openmc.ZCylinder(surface_id=5, x0=0, y0=0, r=0.4)
+fuel2 = openmc.ZCylinder(surface_id=6, x0=0, y0=0, r=0.3)
+fuel3 = openmc.ZCylinder(surface_id=7, x0=0, y0=0, r=0.2)
+
+left.boundary_type = 'vacuum'
+right.boundary_type = 'vacuum'
+top.boundary_type = 'vacuum'
+bottom.boundary_type = 'vacuum'
+
+# Instantiate Cells
+cell1 = openmc.Cell(cell_id=1, name='Cell 1')
```

Comment:
Just want to confirm, that it's expected that `Cell 1` here is upper-cased `C` whereas the other cells are lower-cased `c`.